controllers/statsController.js

```diff
@@ -156,4 +156,18 @@ module.exports = {
       });
     });
   },
+  getRecentClaims (startDate) {
+    logger.debug('retrieving most recent claims');
+    return new Promise((resolve, reject) => {
+      // get the raw requests data
+      db.sequelize.query(`SELECT * FROM File WHERE nsfw != 1 AND trendingEligible = 1 ORDER BY createdAt DESC LIMIT 25;`, { type: db.sequelize.QueryTypes.SELECT })
```

Comment:
Try to avoid SQL in your controllers and keep it located in the modal layer. This helps facilitates future modal refactoring/redesign and keeps controllers simpler (controllers are typically then thinnest of the 3 MVC layers).

Reply:
moved to model https://github.com/lbryio/spee.ch/commit/4d94acee77cd90341f23ab5d39d2ba90bc7edf2c